src/components/documentation/page/page-asyncapi.html

```diff
@@ -0,0 +1,26 @@
+<!--
+
+    Copyright (C) 2015 The Gravitee team (http://gravitee.io)
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+            http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+
+-->
+
+<div ng-if="!$ctrl.pageContent">
+  <gv-icon shape="text:align-center" style="--gv-icon--s: 96px"></gv-icon>
+  <div>No content to display</div>
+</div>
+
+<div ng-if="$ctrl.pageContent">
+  <asyncapi-component schema="{{$ctrl.pageContent}}" cssImportPath="assets/style/asyncapi-component.css"> </asyncapi-component>
```

Comment:
div can be merged into asyncapi-component I think
```html
<asyncapi-component ng-if="$ctrl.pageContent" schema="{{$ctrl.pageContent}}" cssImportPath="assets/style/asyncapi-component.css"> </asyncapi-component>
```

Reply:
done